z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -1,28 +1,28 @@
 # IBM® Cloud Infrastructure Center
 
-IBM® Cloud Infrastructure Center is an advanced infrastructure management offering, including on-premises cloud deployments of IBM z/VM®-based and KVM based Linux® virtual machines on the IBM Z® and IBM LinuxONE platforms.
+IBM® Cloud Infrastructure Center is an advanced infrastructure management product, providing on-premises cloud deployments of IBM z/VM®-based and KVM based Linux® virtual machines on the IBM Z® and IBM LinuxONE platforms.
```

Comment:
Thank you, Stev. However, I pick up this line from our official doc: https://www.ibm.com/docs/en/cic/1.1.3?topic=overview

Reply:
I don't know if this conversation was resolved, but a PR was made so I assume it was. You can also perform PRs between the your team.